repos/fdbt-site/src/pages/selectCapValidity.tsx

```diff
@@ -81,11 +96,12 @@ export const getFieldset = (errors: ErrorInfo[], endOfFareDay?: string): RadioCo
     return CapValidityFieldSet;
 };
 
-const CapValidity = ({ errors = [], fieldset, csrfToken }: CapValidityProps): ReactElement => {
+const CapValidity = ({ capExpirySelected, errors = [], fieldset, csrfToken }: CapValidityProps): ReactElement => {
     return (
         <TwoThirdsLayout title={title} description={description} errors={errors}>
             <CsrfForm action="/api/capValidity" method="post" csrfToken={csrfToken}>
                 <>
+                    {!capExpirySelected ? <BackButton href="/viewCaps" /> : null}
```

Comment:
I think the back button should always appear. User's wont always think that clicking continue would do the same thing.